<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowDevolvendo</class>
 <widget class="QMainWindow" name="MainWindowDevolvendo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>273</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Devolver veiculo</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCheckBox" name="checkBoxSituacaoLocacao">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>179</y>
      <width>57</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Locado</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>66</y>
      <width>27</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Data:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonDevolverLocacao">
    <property name="geometry">
     <rect>
      <x>309</x>
      <y>13</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Devolver</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>10</y>
      <width>80</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Placa do veiculo:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPlacaLocacao">
    <property name="geometry">
     <rect>
      <x>117</x>
      <y>10</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>38</y>
      <width>80</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nome do cliente:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>309</x>
      <y>53</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancelar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Kilometragem na Devolução:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditValorLocacao">
    <property name="geometry">
     <rect>
      <x>118</x>
      <y>94</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>94</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Valor da locação:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditNomeClienteLocacao">
    <property name="geometry">
     <rect>
      <x>117</x>
      <y>38</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditDataLocacao">
    <property name="geometry">
     <rect>
      <x>64</x>
      <y>66</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditKmFinal">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>150</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Kilometragem na locação:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditKmInicial">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>120</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>430</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindowDevolvendo</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>351</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>355</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
